policy/skills/parse_test.go

```diff
@@ -86,6 +86,16 @@ func TestParseIntArgs(t *testing.T) {
 			arg:  int64(30),
 			want: 30,
 		},
+		{
+			name: "Parse int32 arg",
+			arg:  int32(30),
+			want: 30,
+		},
+		{
+			name: "Parse float64 arg",
+			arg:  float64(30),
+			want: 30,
+		},
```

Comment:
Is this case to cover the hypothetical situation where a user specifies a float number as the age of vulnerabilities? e.g. `15.2`

Reply:
I don't remember, but the implementation had support for `float64` so I just added the test to match.